app/views/layouts/application.html.erb

```diff
@@ -12,7 +12,8 @@
 
   <body>
     <nav>
-      <%= link_to 'Home', root_path %>
+      <%= image_tag("logo.png", width: 50) %>
```

Comment:
My buddy Cecile made a logo for me and I wanted to at least put it in somewhere!